### DISCUSSION TITLE
How would I use the classic style of app icon set for iOS?

### DISCUSSION THREAD

**Discussion by AnthonyIacono** [dotnet/maui#12381] [Q&A] [answered]
I'm running into an issue with specifying my app icon. For context, I am converting an old Xamarin Forms application to .NET MAUI, and I have an `Assets.xcassets` already. I **do not** want to use `MauiIcon`. What am I doing wrong here?

I have an `Info.plist` inside `Platforms/iOS/Info.plist` and the following lines in it:
```
<key>XSAppIconAssets</key>
<string>Assets.xcassets/appicon.appiconset</string>
```

I have the `Assets.xcassets` file in the root directory. I have tried also copying it into `Platforms/iOS/Assets.xcassets`, but no matter what I do I am greeted with the default "no icon" in the simulator when building.

I have removed `MauiIcon` from the `csproj` as well.

What am I doing wrong? Thanks!

**Comment by AnthonyIacono** [comment, marked as answer]
Alright, I solved it! You must add the `Assets.xcassets` as a `BundleResource` in your `csproj`.

For example:

```
<BundleResource Include="Assets.xcassets\**\*" />
```

After that, you simply specify the path inside your `plist` as seen above.

**Reply by PRNDA** [reply]
Sorry, but this doesn't work for me.
After compilation, the key `<key>XSAppIconAssets</key>` was removed from the final Info.plist file, leaving me a blank app icon on the screen.

**Reply by taublast** [reply]
Confirmed to be working with net9.

files:

Placed as `original.appiconset` (do not conflict with resizetizer-generated `appiconset.appiconset`) into "Platforms\iOS\Assets.xcassets". Do not include anything manually via UI, do not drag and drop anything, just place files in that location outside Visual Studio. 

csproj:

```xml
  <ItemGroup Condition="$(TargetFramework.Contains('ios')) == true">
    <BundleResource Include="Platforms\iOS\Assets.xcassets\**\*" />
  </ItemGroup>
```

plist:

```xml
	<key>XSAppIconAssets</key>
	<string>Assets.xcassets/original.appiconset</string>
```
